source/target/index.html

```diff
@@ -0,0 +1,17 @@
+<html>
+    <link href="animation.css" rel="stylesheet">
+    <div class="loader loader--style2" style="color: #FF6700;" title="1">
```

Comment:
If the transition issue persists, we can add it as a task and put it on the backlog for this sprint.

Reply:
I agree.